00-short-introduction-to-Python.md

```diff
@@ -244,6 +244,85 @@ or
 >>>
 ```
 
+## Syntax: Indentation and Code Blocks
+
+In Python, differently from other programming languages, there are no specific
+markers for code blocks. A code block  is defined by the text indentation of the
+block itself.
+
+A code block section starts by a Python `if`, `for`, etc. statement, or a function
+or class definition, `def`,  followed by a period `:`. The code block to which the
+statement or definition refers to, it is indented by at least one space more than
+the initial statement itself. It is the indentation  what marks the following code
```

Comment:
"it is the indentation what marks the following code block extension" > rephrase